buildpipeline/Core-Setup-Signing-Windows-BT.json

```diff
@@ -489,7 +475,7 @@
         "logProjectEvents": "false",
         "createLogFile": "false",
         "msbuildLocationMethod": "version",
-        "msbuildVersion": "15.0",
+        "msbuildVersion": "latest",
```

Comment:
I've been moving them all to latest in this and other pr's.  specifying 15.0 causes a warning and then it just defaults to "latest".  This caused noise that bothered a bunch of people that look at results on mission control.